package.json

```diff
@@ -61,19 +67,14 @@
     "redux-persist": "^5.10.0",
     "redux-thunk": "^2.3.0",
     "shortid": "^2.2.13",
-    "uuid": "^3.3.2",
-    "yup": "^0.26.10",
-    "chai": "4.2.0",
-    "dotenv": "6.2.0",
-    "mocha": "5.2.0",
-    "wdio-chromedriver-service": "0.1.5",
-    "randomstring": "1.1.5",
-    "random-number": "0.0.9",
     "ts-node": "^7.0.1",
     "tsconfig-paths": "^3.7.0",
     "tslint": "^5.12.1",
     "typescript": "^3.2.2",
-    "webdriverio": "^5.3.0"
```

Comment:
@aljvilla please update just the line you need to add for the `package.json`. Remove oťher changes.